addons/core/translations/resources/en-us.yaml

```diff
@@ -218,6 +218,17 @@ host-catalog:
     aws: AWS
     azure: Azure
     plugin: Dynamic
+    credential:
+      label: Credential type
+      help: Select the credential type that you want to use.
+      static-credential:
+        edit: Static credentials
```

Comment:
Should we be capitalizing `credentials` for this and dynamic or no?

Reply:
hmmm, the design did not have upper case for creds, @madeleinetle  thoughts?

Reply:
Yeah, we should probably capitalize them. Just updated [the design](https://www.figma.com/design/YO4WntE8sIHT7oThkB8fQf/Support-Assume-role-for-AWS-DHC?node-id=2908-27909&t=LapycksBRufEHNpR-1) to reflect this change